editor/editor_settings.cpp

```diff
@@ -1612,7 +1617,7 @@ void EditorSettings::set_builtin_action_override(const String &p_name, const Arr
 	}
 
 	// Update the shortcut (if it is used somewhere in the editor) to be the first event of the new list.
-	if (shortcuts.has(p_name)) {
+	if (shortcuts.has(p_name) && !event_list.is_empty()) {
 		shortcuts[p_name]->set_event(event_list.front()->get());
 	}
```

Comment:
Hmm... I am pretty sure that in this case we want to remove the shortcut event in the case that the event list is empty. Otherwise shortcuts in the editor will not be properly reflecting the true state of the settings.

```suggestion
	if (shortcuts.has(p_name)) {
		shortcuts[p_name]->set_event(event_list.is_empty() ? Ref<InputEvent>() : event_list.front()->get());
	}
```

Reply:
@EricEzaM 
Thank you for suggestions. If your okay with that, i can make a single commit from suggestions.

Reply:
You would still need to squash it afterwards, so it doesn't matter how you apply the suggestions 🙃

Reply:
Yep, for sure all into one commit :)